### DISCUSSION TITLE
Search of page metadata including author, categories, date, etc.

### DISCUSSION THREAD

**Discussion by madisonostermann** [quarto-dev/quarto-cli#10479] [Feature Requests]
### Description

I didn't see an open issue or discussion thread mentioning this, and haven't seen any documentation, but please point it out if I missed it!

My team is building our knowledge base in a Quarto website and searchability is one of the most highly used features. The built-in search capability (we haven't used Algolia) surfaces content great, but it would be wonderful to also make the metadata at the top of each page searchable, e.g. to find all pages authored by a specific team member who is about to offboard, to find any listings with certain category tags, etc.

**Comment by mcanouil** [comment]
Have you tried configuring Algolia?
See <https://quarto.org/docs/websites/website-search.html#using-algolia>.

In particular, https://quarto.org/docs/websites/website-search.html#advanced-configuration
